Description: Usage of copyright, prolog, & epilog files. 
 - Include copyright, prolog, and epilog in all segments.
 - Also, bump version to 3.3.1
Origin, upstream: http://makenl.cvs.sourceforge.net/viewvc/makenl/makenl/src/output.c?revision=1.7
Origin, vendor: http://sourceforge.net/p/makenl/code, acb21c6153c4c9fc97ca009d6b4662d74589b4b7
Author: Andrew Leary <ajleary@users.sourceforge.net>
Last-Update: 2012-12-26

---
 docs/history.txt |   11 ++++++++++-
 src/output.c     |    7 ++++---
 src/version.h    |    4 ++--
 3 files changed, 16 insertions(+), 6 deletions(-)

diff --git a/docs/history.txt b/docs/history.txt
index 8b1b113..bf3f653 100644
--- a/docs/history.txt
+++ b/docs/history.txt
@@ -1,8 +1,17 @@
-/*  $Id: history.txt,v 1.15 2012/11/12 08:45:47 ajleary Exp $  */
+/*  $Id: history.txt,v 1.16 2012/12/26 04:21:07 ajleary Exp $  */
 
 
                             MakeNL Revision History
 
+       v3.3.1 -- Dec 25, 2012
+
+            Include Copyright, Prolog, and Epilog files in all nodelist
+            segments, not just in composite nodelists.  The filenames (by
+            default) are cpyright.txt, prolog.txt, and epilog.txt in your
+            master directory.  They can be overridden with the COPyright,
+            PROlog, and EPIlog statements in your makenl.ctl file.
+            -- Andrew Leary
+
        v3.3.0 -- Nov 12, 2012
 
             Release version of v3.2.19 for wide distribution.
diff --git a/src/output.c b/src/output.c
index ec3e26b..4e57132 100644
--- a/src/output.c
+++ b/src/output.c
@@ -1,4 +1,4 @@
-/* $Id: output.c,v 1.6 2012/10/16 09:15:51 ozzmosis Exp $ */
+/* $Id: output.c,v 1.7 2012/12/26 04:21:08 ajleary Exp $ */
 
 #include <stdio.h>
 #include <stdlib.h>
@@ -85,9 +85,10 @@ CopyComment(FILE * output, char *Copyfile, const char *year,
     char fullname[MYMAXDIR];
     char linebuf[linelength];
 
+    /* Removed 12/12/12 */
     /* Don't include copyright, prolog and epilog unless we make the composite list */
-    if (MakeType != LEVEL_TOP)
-        return lineno;
+    /* if (MakeType != LEVEL_TOP)
+        return lineno; */
     if (!Copyfile)
         return lineno;
     myfnmerge(fullname, NULL, MasterDir, Copyfile, NULL);
diff --git a/src/version.h b/src/version.h
index 4e887a0..72941b7 100644
--- a/src/version.h
+++ b/src/version.h
@@ -1,8 +1,8 @@
-/* $Id: version.h,v 1.15 2012/11/12 08:45:48 ajleary Exp $  */
+/* $Id: version.h,v 1.16 2012/12/26 04:21:08 ajleary Exp $  */
 
 #include "os.h"
 
-#define MAKENL_VERSION      "3.3.0"
+#define MAKENL_VERSION      "3.3.1"
 
 #define MAKENL_LONG_VERSION "MakeNL " MAKENL_VERSION " (" MAKENL_OS ") compiled with " MAKENL_CC " on " __DATE__ " " __TIME__
 
-- 
1.7.10.4

